woothemes/woocommerce-rest-api-docs
# Add duplicate product docs

### Changes proposed in this Pull Request:

The WooCommerce PR https://github.com/woocommerce/woocommerce/pull/46141 added a new REST API endpoint meant to `duplicate a product`.

This PR adds the documentation necessary for that new endpoint

It adds the documentation under `v3` and under `wp-api-v3`.

### How to test the changes in this Pull Request:

1. Checkout this branch and build the project
```
sh build.sh
```
2. After running the build process, you will see a `build` folder in your project.
3. Go to the `build` folder and open the following files in your browser:
- `./index.html`
- ~`./v3.html`~ (updated @mattsherman - our change is only for the WP API, not the legacy API)
4. Go to `Product > Duplicate product` in the sidebar.
5. Verify that the documentation looks correct.
